### ISSUE TITLE
FEEDBACK: Submitter privilege control

### ISSUE THREAD

**Issue by oopsliu** [tedrick/Survey123-Testing#234]
With version 3.5 of Survey123 website we have added 2 settings to control the privilege of submitter on the **Collaborate** tab: 

- "What can submitter do" section will control the capability of submitter, whether the submnitter can add, update or delete their own records:

![image](https://user-images.githubusercontent.com/7564300/59020072-d193f380-887b-11e9-8fd7-56283998edf6.png)

___
 - "Multiple submissions" section allows you to decide whether your survey allows the same user to respond to your survey multiple times, or if only one response is permitted. Usernames will be used to identify signed-in users, and browser cookies are used for anonymous respondents.

![image](https://user-images.githubusercontent.com/7564300/59017755-e7eb8080-8876-11e9-999f-24cfe88aa35f.png)

> Note: This feature is currently only available in the Survey123 web app. Limiting your survey to one response per user is not honored by the Survey123 field app.

Please comment with your observations / feedback

**Comment by BrandonArmstrong** [comment]
Jerry

**Comment by fandrade86** [comment]
Survey Submitted Already was successful. After I allowed for multiple submissions, I was able to successfully submit.

![SurveySubmittedAlready](https://user-images.githubusercontent.com/21234095/59069891-97bbf080-886d-11e9-8ffd-3acfb6046b31.png)

**Comment by gisguy007** [comment]
Enhancement: Update the "Send Error" message to include additional error reasons. 
Set-up: For the "What can submitters do?", the "Only add new records" was chosen.  When the test submitter (Brandon) tried to edit and resend he got the following message: ![image](https://user-images.githubusercontent.com/43858983/59058673-01c69c80-8852-11e9-9220-63e189348b45.png)

The Multiple submissions and Survey status worked (thank you Francisco for helping me test- he was added to the testers list when I changed the "Who" to everyone (Public).
